pkg/rtc/mediatrack.go

```diff
@@ -96,6 +96,7 @@ func NewMediaTrack(track *webrtc.TrackRemote, params MediaTrackParams) *MediaTra
 
 	if params.TrackInfo != nil && t.Kind() == livekit.TrackType_VIDEO {
 		t.UpdateVideoLayers(params.TrackInfo.Layers)
+		// LK-TODO: maybe use this or simulcast flag in TrackInfo to set simulcasted here
```

Comment:
Made this note and one below to set `simulcasted` flag based on protocol fields rather than the guess work. Can we do this?